website/client/src/assets/scss/form.scss

```diff
@@ -280,63 +282,110 @@ $bg-disabled-control: #34303a;
   padding-left: 36px;
 }
 
+//radio buttons
```

Comment:
Adding comments to our SCSS files is probably good practice! Let's settle on a style, though. You've got three different formats, here:
``// comment``
``//comment``
``//    // comment``

I think I prefer the first!

Reply:
comments standardized